### PR TITLE
fix #75859 ww.animesanka.com

### LINKED ISSUE #75859: ww.animesanka.com
### Issue URL (Anti Adblock Script)
 [https://ww.animesanka.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fww.animesanka.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads, <br/>EasyList<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Fanboy's Annoyances, <br/>Web Annoyances Ultralist, <br/>Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion, <br/>BarbBlock<br/><br/><b>Language-specific:</b><br/> AdGuard French, <br/>AdGuard Russian, <br/>Liste AR

### PR DESCRIPTION
**Pull request by wout-codes**
#75859

### CODE REVIEW

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -33,6 +33,16 @@ video.gjirafa.com#%#AG_defineProperty('originVideo.requestA.Id', { value: "0" })
...
+animesanka.com#%#//scriptlet("prevent-addEventListener", "load", "n")
+!#if (adguard_ext_safari || adguard_app_ios || adguard_ext_android_cb)
+animesanka.com#@##DivTopAd
+animesanka.com#@##tobsideAd
+animesanka.com#@##leftframeAD
+animesanka.com#@##iqadtile5
+animesanka.com#@##ads-sticky
+animesanka.com#@##ad_190x90
+animesanka.com#@##ad-zone-1
+!#endif
```
Comment (zloyden) [inline EnglishFilter/sections/foreign.txt:45]: 1. Remove your rules.
2. Add domain to the common rules to the section `g207`. `#%#//scriptlet('abort-current-inline-script', 'document.addEventListener', '/google_ad_client/')`.
3. Also add domain to the `content_blockers.txt`. We have a section for `g207`, just `@@||animesanka.com^$generichide`.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/foreign.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/foreign.txt:45]: Haha.. I did everything wrong then.

**Review by zloyden — APPROVED** [review]

File: EnglishFilter/sections/foreign.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/foreign.txt]: Useless rules. I know something that you do not.

Comment (zloyden) [reply EnglishFilter/sections/foreign.txt]: What do you mean here https://github.com/AdguardTeam/AdguardFilters/pull/75879#discussion_r583719582?

Comment (wout-codes) [reply EnglishFilter/sections/foreign.txt]: These rules doesn't even work! I tried about 2 hours ago.

Comment (AdamWr) [reply EnglishFilter/sections/foreign.txt]: Which rules and where it doesn't work?

Comment (wout-codes) [reply EnglishFilter/sections/foreign.txt]: These rules doesn't work anymore. Please check, I am tired. https://github.com/AdguardTeam/AdguardFilters/issues/75931

Comment (AdamWr) [reply EnglishFilter/sections/foreign.txt]: This rule `animesanka.com#%#//scriptlet('abort-current-inline-script', 'document.addEventListener', '/google_ad_client/')` works fine on my end.
There is a different domain in #75931, there is `animesanka.club` not `animesanka.com` and adding:
`animesanka.club#%#//scriptlet('abort-current-inline-script', 'document.addEventListener', '/google_ad_client/')`
also works fine on my end.

Comment (zloyden) [reply EnglishFilter/sections/foreign.txt]: Good catch about the domain, @AdamWr :)

Comment (wout-codes) [reply EnglishFilter/sections/foreign.txt]: @AdamWr I have tried that with asterisk before saying useless no offense. The only working scriptlet on my end is `animesanka.*#%#//scriptlet("prevent-addEventListener", "load", "n")`.

**Pull request merged**